test/functional/p2p_addrv2_relay.py

```diff
@@ -63,14 +63,12 @@ def run_test(self):
         addr_receiver = self.nodes[0].add_p2p_connection(AddrReceiver())
         msg.addrs = ADDRS
         with self.nodes[0].assert_debug_log([
-                'Added 10 addresses from 127.0.0.1: 0 tried',
                 'received: addrv2 (131 bytes) peer=0',
-                'sending addrv2 (131 bytes) peer=1',
+                'sending addrv2 (14 bytes) peer=1',
```

Comment:
Wrote yesterday on discord, but rewriting here so we don't forget:
Imo the assert here should not be changed because we are functional testing a specific expected behaviour. If this change is due to the fact that `m_addr_token_bucket` is empty then it is better if you change the node time to refill it, if it's not that then the PR introduced a bug